src/promptflow/promptflow/_cli/_pf/entry.py

```diff
@@ -58,6 +61,14 @@ def entry(argv):
         if hasattr(args, "debug") and args.debug:
             for handler in logging.getLogger(LOGGER_NAME).handlers:
                 handler.setLevel(logging.DEBUG)
+
+        activity_name = f"pf.{args.action}.{args.sub_action}.cli"
+        message = f"{activity_name}.start"
+        add_telemetry_log(activity_name=activity_name,
```

Comment:
Directly call `with log_activity()` here? so you don't need to dup the logging logic in 2 functions?

Reply:
done